### ISSUE TITLE
This is such a method of solving issues - to close them? NOT solving?

### ISSUE THREAD

**Issue by keva1** [omniedgeio/omniedge#58]
This is such a method of solving issues - to close them? NOT solving?

**Comment by keva1** [comment]
Close your crooked app better if you haven't learned how to communicate normally with people...

**Comment by yongqianme** [comment]
As I said very friendly to you, there was a problem between your windows PC to API, something related to your network, not the service from omniedge. That’s why the issue was closed. I am not in a position to fix your network problem. 

You may try iOS , android, Cli or macOS client to compare.

**Comment by yongqianme** [comment]
And if your don’t mind, leave your account information here, we can help to double check if we can login successfully.

**Comment by keva1** [comment]
Issues are considered closed WHEN THEY ARE RESOLVED! Isn't it? I haven't heard 1 proposal for a solution. You yourself your behavior does not resemble prohibitions on the Internet you know from whom??

**Comment by keva1** [comment]
nin-05@mail.ru

**Comment by keva1** [comment]
Realy -  nin-05@tut.by

**Comment by keva1** [comment]
I hope it's not for me to just be banned...

**Comment by keva1** [comment]
What do you see there?

**Comment by yongqianme** [comment]
To test your account login , we will need time to prepare a clean windows, which takes time. Before that, you can try android , iOS, macOS or Cli to compre.

**Comment by yongqianme** [comment]
Without your password, we can not test your account. You can send email with your password  to support@omniedge.io. To be honest, it will not the account issue.

**Comment by keva1** [comment]
I sent it to the support@omniedge.io.

**Comment by keva1** [comment]
Android over Wi-Fi on the same network as the PC connected without problems and received an IP. So the network is fine...

**Comment by yongqianme** [comment]
Where are your windows PC physically located by the way ?

**Comment by yongqianme** [comment]
> Android over Wi-Fi on the same network as the PC connected without problems and received an IP. So the network is fine...

So Your network to omniedge service is okay. The only problem is your windows PC. Would you try run OmniEdge windows with admin ?

**Comment by keva1** [comment]
I do not understand the question. My home address? If the smartphone connected via Wi-Fi from the same network with the PC, then I have an order with the network. Is not it?

**Comment by keva1** [comment]
I run omni with admin rights. Now I repeated it again. But it didn't help...

**Comment by yongqianme** [comment]
> I do not understand the question. My home address? If the smartphone connected via Wi-Fi from the same network with the PC, then I have an order with the network. Is not it?

Country is enough. I would like to use the same location to test if AWS has the data center in your country.

**Comment by keva1** [comment]
But from the phone, everything works on Wi-Fi. My country is BELARUS

**Comment by keva1** [comment]
We don't have DT AWS

**Comment by yongqianme** [comment]
It is the issue of windows client, new issue is opened. Our programmer will take care. 
Thank you very much for submitting the issue. 

https://github.com/omniedgeio/omniedge/issues/59

**Comment by yongqianme** [comment]
The issue is solved, try again. 
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/93888/203527455-56c71879-fcb6-4db1-acdd-396827a090e6.png">

**Comment by keva1** [comment]
Yes, Windows clients have been added. Thank you! What was the problem? BUT! Added clients do not ping each other, as I wrote at the very beginning. How to fix it?

**Comment by yongqianme** [comment]
Regarding the ping, there are two ways you can help yourself. 
1. Setup a self-hosted supernode which is close to you, the public supernode for BELARUS is in the US which makes the auth slow.  Your FREE plan is also supported !! Instruction is here: https://omniedge.io/docs/article/install/customize-supernode
2. If the ping didn't work only on windows, check: https://omniedge.io/docs/article/Others#why-cant-ping-my-windows-pc

**Comment by keva1** [comment]
1. What about the Frankfurt,DE server is not available? Do you know where Belsrus is located?
2. Naturally, ping is allowed in the Windows Firewall rules. But the smartphone on Android does NOT respond either. And your Amazon server ... From the phone, too, not 1 PC responds. Just the phone itself...

**Comment by yongqianme** [comment]
1. Your account will be allocate a public supernode automatically, normally a public supernode in the USA. Your own self-hosted will provide better experience. It is the reason we offer this feature to FREE users like you. 
2. I just tested with a AWS VPS running cli, and ping form the windows from the test windows works well. 
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/93888/203656189-9a06a156-763f-4e04-b4df-2fd649ac0127.png">

**Comment by keva1** [comment]
1. How are servers distributed automatically? In my opinion, it is logical to redirect to the nearest server in Frankfurt and not to the other side of the globe. Or how does this autodetection work?
2. Linux machine and I pinged from my smartphone. A NO 1 Windows client - no!
![photo_2022-11-24_01-35-36](https://user-images.githubusercontent.com/65159864/203657693-9c3ed618-0c64-4cab-9d32-05b61ea2a20e.jpg)

**Comment by yongqianme** [comment]
From my side, the ping from the linux device to EC2 windows is not work, but ping from EC2 windows to linux works. It is because the ping response is default disabled. 
You can try ping from your PC to your android or the linux device. I will keep the test VPS for 2 hours for your test. 

By the way, I can not ping your android phone from the the linux device or EC2 windows.

**Comment by keva1** [comment]
So I write about that and that Windows devices do not respond. I configured a firewall on these 2 machines on icmp. And you can make a ping screen on a smartphone, like mine. He is the most visible.

**Comment by keva1** [comment]
And I did not understand about the nearest server. Or do you really have no servers in Europe?

**Comment by keva1** [comment]
1. So it is NOT right for Europe to allocate a server in the USA. WHY then a server in Germany?
2. For me most interested in Windows PC, as working...

**Comment by keva1** [comment]
It is clear that there are no servers in either Frankfurt or Milan.
Here is my screen.
![net](https://user-images.githubusercontent.com/65159864/203823568-ee2ad141-c4b0-4f1b-9cc7-ed28c30f991c.png)

**Comment by keva1** [comment]
It is common practice for ALL network services to work through the nearest server. Otherwise, for $3-5 I will set up my VPS with WG or SS, Netmaker or Nebula, and why do I need an omni otherwise?

**Comment by keva1** [comment]
So me with a single supernode in the US of yours, so you don't pretend there, then why should I? And with jambs with clients on Windows ...

**Comment by keva1** [comment]
WHERE did I write about VPN? I know what an intranet is, if anything ... Explain to me, as a user, why a node in the US for those in Europe ?
